static/src/javascripts/types/global.d.ts

```diff
@@ -168,6 +168,8 @@ interface Window {
 		mustardCut?: boolean;
 		polyfilled?: boolean;
 		adBlockers: AdBlockers;
+		// /frontend/common/app/templates/inlineJS/blocking/enableStylesheets.scala.js
```

Comment:
You may want to add a permalink, in case this file changes: https://github.com/guardian/frontend/blob/5b970cd7308175cfc1bcae2d4fb8c06ee13c5fa0/common/app/templates/inlineJS/blocking/enableStylesheets.scala.js#L13-L16